backend/routes/bookingRoutes.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
```

Comment:
You can add the same library as in the adminRoutes. Also, establish connectivity with database

```javascript
const { body, validationResult } = require('express-validator');
const pool = require('../db'); // Import the database connection
```